etld/domain.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2018 The Brave Software Team. Distributed under the MPL2
```

Comment:
nit: Should be 2019. This applies to all of the new files added in this PR.

Reply:
fixed :)

Reply:
Maybe I'm confused about how GitHub is displaying the latest version of your PR, but it seems like you fixed all files, except for `etld/domain.cc`?

Reply:
No, i goofed, apologies for the butter fingers.  Fixed now